task/github-create-deployment/0.1/github-create-deployment.yaml

```diff
@@ -96,7 +102,8 @@ spec:
         import os
         import sys
 
-        deployment_url = "/repos/$(params.REPO_FULL_NAME)/deployments"
+        deployment_url = "$(params.API_PATH_PREFIX)" + \
```

Comment:
hi @devholic, I see that there are changes in the script. Can we please bump the version of the task?

Reply:
Sure 🙂